packages/plugin_platform_interface/pubspec.yaml

```diff
@@ -12,12 +12,12 @@ description: Reusable base class for Flutter plugin platform interfaces.
 # be done when absolutely necessary and after the ecosystem has already migrated to 1.X.Y version
 # that is forward compatible with 2.0.0 (ideally the ecosystem have migrated to depend on:
 # `plugin_platform_interface: >=1.X.Y <3.0.0`).
-version: 1.0.3
+version: 2.0.0-nullsafety
 
 repository: https://github.com/flutter/plugins/tree/master/packages/plugin_platform_interface
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: '>=2.10.0-56.0.dev <3.0.0'
```

Comment:
I think making SDK min dep to a dev version will cause pub to warn when publishing. If that causes CI problem we might need to somehow ignore it.

Reply:
this is a moving target. See https://dart.dev/null-safety#configure-the-sdk-version for more.

Reply:
Sure. Just wanted to point it out, let's see if CI passes.

Reply:
It's passing